### PR TITLE
Refactor user and TO commands into separate files

### PR DESCRIPTION
**Pull request by kev306**
closes issue #434

### CODE REVIEW

**Comment by kev306** [comment]
Still in progress

**Review by vck3000 — APPROVED** [review]

File: src/sockets/commands/user/help.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    for (const key in userCommands) {
+      if (userCommands.hasOwnProperty(key)) {
+        const commandKey = key as keyof typeof userCommands;
+        dataToSend.push({
+          message: userCommands[commandKey].help,
+          classStr: 'server-text',
+        });
+      }
+    }
```
Comment (vck3000) [inline src/sockets/commands/user/help.ts:19]: Looks good! Let's remove the below comments.

File: src/sockets/commands/user/index.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import { pausetimer } from './pausetimer';
+import { unpausetimer } from './unpausetimer';
+import { voidgame } from './voidgame';
+
+export const userCommandsImported: Commands = {
+  [help.command]: help,
+  [navbar.command]: navbar,
+  [pausetimer.command]: pausetimer,
+  [roll.command]: roll,
+  [unpausetimer.command]: unpausetimer,
```
Comment (vck3000) [inline src/sockets/commands/user/index.ts:14]: Let's put the pause timer and unpause timer together.

File: src/sockets/commands/user/roll.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Command } from '../types';
+import { SocketUser } from '../../types';
+import { sendReplyToCommand } from '../../sockets';
+
+export const roll: Command = {
+  command: 'roll',
+  help: '/roll <optional number>: Returns a random number between 1 and 10 or 1 and optional number.',
+  run: async (args: string[], socket: SocketUser) => {
+    console.log(args.length);
```
Comment (vck3000) [inline src/sockets/commands/user/roll.ts:9]: Let's remove this console log

File: src/sockets/commands/user/roll.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      sendReplyToCommand(socket, 'Enter just one number.');
+      return;
+    }
+    if (args[1]) {
+      const optionalNumber = parseInt(args[1]);
+
+      if (isNaN(optionalNumber) || optionalNumber.toString() !== args[1]) {
+        sendReplyToCommand(socket, 'That is not a valid number!');
+        return;
+      } else {
```
Comment (vck3000) [inline src/sockets/commands/user/roll.ts:20]: Let's remove the else because you have an early return above already.

File: src/sockets/sockets.ts [hunk 5]
```diff
@@ -409,19 +371,18 @@ export const userCommands = {
...
   },
 
   interactUser: {
     command: 'interactUser',
     help: '/interactUser <buzz/pat/poke/punch/slap/hug> <playername>: Interact with a player.',
-    run(data, senderSocket) {
-      console.log('interact user', data);
-      const { args } = data;
+    run(args: string[], senderSocket) {
+      console.log('interact user', args);
```
Comment (vck3000) [inline src/sockets/sockets.ts:385]: Let's remove this line

**Pull request merged**